From 3bda26dab5de6c5e2490e6424d51555f30b4c2da Mon Sep 17 00:00:00 2001
From: Sujeet Baranwal <sujeet.baranwal@cavium.com>
Date: Tue, 27 Nov 2018 04:01:12 +0300
Subject: [PATCH 0782/1051] dt-bindings: mmc: Add emmc bus pin output parameter

Add parameter for programming board dependent emmc
bus pin output drive and slew rate control

Signed-off-by: Sujeet Baranwal <sujeet.baranwal@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 Documentation/devicetree/bindings/mmc/cavium-mmc.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/mmc/cavium-mmc.txt b/Documentation/devicetree/bindings/mmc/cavium-mmc.txt
index 1433e6201dff..580fe086638b 100644
--- a/Documentation/devicetree/bindings/mmc/cavium-mmc.txt
+++ b/Documentation/devicetree/bindings/mmc/cavium-mmc.txt
@@ -21,6 +21,8 @@ Optional properties:
    please refer to mmc.txt within this directory
  - cavium,cmd-clk-skew : number of coprocessor clocks before sampling command
  - cavium,dat-clk-skew : number of coprocessor clocks before sampling data
+ - cavium,drv-strength : eMMC bus pin output drive strength
+ - cavium,clk-slew     : eMMC bus pins output slew rate control
 
 Deprecated properties:
 - spi-max-frequency : use max-frequency instead
-- 
2.17.1

